datatypes/papas/DefinitionsCollections.h

```diff
@@ -0,0 +1,42 @@
+//
+//  DefinitionsCollections.h
+//  papas
+//
+//  Created by Alice Robson on 26/05/16.
+//
+//
+
+#ifndef DefinitionsCollections_h
+#define DefinitionsCollections_h
+
+#include "Id.h"
+//#include <map>
+#include <unordered_map>
+#include <vector>
+namespace papas {
+class Cluster;
```

Comment:
Can you check my message on skype? 

Le 13 sept. 2016 à 12:57, alicerobson <notifications@github.com> a écrit :

> In datatypes/papas/DefinitionsCollections.h:
> 
> > +//  DefinitionsCollections.h
> > +//  papas
> > +//
> > +//  Created by Alice Robson on 26/05/16.
> > +//
> > +//
> > +
> > +#ifndef DefinitionsCollections_h
> > +#define DefinitionsCollections_h
> > +
> > +#include "Id.h"
> > +//#include <map>
> > +#include <unordered_map>
> > +#include <vector>
> > +namespace papas {
> > +class Cluster;
> I think this is infact the fcc coding standard. See:
> https://github.com/jlingema/FCCSW/blob/master/doc/CppCodingStyleGuidelines.md#indentation
> All the code has been passed through a clang formatter with the fcc clang file.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>